webapp/packages/product-default/vite.config.ts

```diff
@@ -18,6 +18,9 @@ export default defineConfig(
     Fixes https://github.com/dbeaver/cloudbeaver/issues/3308
     ROOT_URI approach for assets doesn't work well when we serve our application not from a root folder. In that case inner Vite asset handling system add indices to CSS files using absolute path ignoring injected ROOT_URI. When using relative path in base property, we ask Vite to generate paths relatively to each file https://vite.dev/guide/build#public-base-path.  */
     base: './',
+    build: {
+      cssCodeSplit: false,
+    },
```

Comment:
it's better to configure it with baseConfigurationPlugin for all products in one place and please add comment why we use this option